impeller/renderer/render_target.cc

```diff
@@ -322,6 +327,17 @@ RenderTarget RenderTarget::CreateOffscreenMSAA(
   color0.texture = color0_msaa_tex;
   color0.resolve_texture = color0_resolve_tex;
 
+  if (context.GetCapabilities()->SupportsImplicitResolvingMSAA()) {
+    // If implicit MSAA is supported, then the resolve texture is not needed
+    // because the multisample texture is automatically resolved. We instead
+    // provide a view of the multisample texture as the resolve texture (because
+    // the HAL does expect a resolve texture).
+    //
+    // In practice, this is used for GLES 2.0 EXT_multisampled_render_to_texture
+    // https://registry.khronos.org/OpenGL/extensions/EXT/EXT_multisampled_render_to_texture.txt
+    color0.resolve_texture = color0_msaa_tex;
```

Comment:
I think we should technically create some sort of "Wrapped" texture with the same descriptor but with sample count set to 1? We might need to expand on what wrapped means. Though if it doesn't cause any problems as is maybe we just leave it.